src/main/java/com/limechain/sync/fullsync/FullSyncMachine.java

```diff
@@ -120,6 +124,7 @@ public void start() {
     private void initializeEpochState() {
         epochState.initialize(runtime.callBabeApiConfiguration());
         epochState.setGenesisSlotNumber(runtime.getGenesisSlotNumber());
+        slotCoordinator.start(List.of(babeService));
```

Comment:
I'd put `AppBean.getBean(BabeService.class)` as a local var here because it wouldn't be used anywhere else in the fullsync machine.